CODIGO/frmOpciones.frm

```diff
@@ -906,15 +906,7 @@ Private Sub chkInvertir_MouseUp(Button As Integer, Shift As Integer, x As Single
     On Error GoTo chkInvertir_MouseUp_Err
     
 
```

Comment:
why remove this? we need to support the setting

Reply:
Resolved in next patch